Zend/zend_object_handlers.h

```diff
@@ -339,7 +339,12 @@ ZEND_API bool ZEND_FASTCALL zend_asymmetric_property_has_set_access(const zend_p
 } while (0)
 
 #define zend_free_trampoline(func) do { \
+		HashTable *attributes = (func)->common.attributes; \
+		if (attributes && !(GC_FLAGS(attributes) & GC_IMMUTABLE) && !GC_DELREF(attributes)) { \
```

Comment:
```suggestion
		if (attributes && !GC_TRY_DELREF(attributes)) { \
```

Reply:
`GC_TRY_DELREF` is `void`. This makes somewhat sense, as the return value would not really be well-defined when the argument is immutable. In any case, changing that would be something for master only.

Reply:
Ah, I missed that. Ofc.

Reply:
Or use zend_array_release